server/storage/mvcc/hash.go

```diff
@@ -86,7 +86,12 @@ type KeyValueHash struct {
 }
 
 type HashStorage interface {
-	// Hash computes the hash of the KV's backend.
+	// Hash computes the hash of the whole backend keyspace,
+	// including key, lease, and other buckets in storage.
+	// This is designed for testing ONLY!
```

Comment:
> This is designed for testing ONLY!

It's align with proto definition. 

REF: https://github.com/etcd-io/etcd/blob/04346e870fc0823d12ca8220c3489635e9e15e72/api/etcdserverpb/rpc.proto#L224